network/p2p/inspector/validation/errors_test.go

```diff
@@ -63,19 +63,20 @@ func TestErrRateLimitedControlMsgRoundTrip(t *testing.T) {
 	assert.False(t, IsErrRateLimitedControlMsg(dummyErr), "IsErrRateLimitedControlMsg should return false for non-ErrRateLimitedControlMsg error")
 }
 
-// TestErrDuplicateTopicRoundTrip ensures correct error formatting for ErrDuplicateTopic.
+// TestErrDuplicateTopicRoundTrip ensures correct error formatting for DuplicateFoundErr.
```

Comment:
Please also add a similar test for `IWantCacheMissThresholdErr`.

Reply:
https://github.com/onflow/flow-go/pull/4574/commits/d9c7970098f3756a6149cd6b6d7044ddccb18a09